CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Analysis/AnalysisViewController.swift

```diff
@@ -244,6 +249,65 @@ import UIKit
                                      overlayView.bottomAnchor.constraint(equalTo: view.bottomAnchor)])
     }
 
+    private func configureLoadingIndicator() {
+        loadingIndicatorView.color = GiniColor(light: .GiniCapture.dark1, dark: .GiniCapture.light1).uiColor()
+
+        addLoadingContainer()
+        addLoadingView(intoContainer: loadingIndicatorContainer)
+
+        if let loadingIndicator = giniConfiguration.analysisScreenLoadingIndicator {
+            addLoadingText(below: loadingIndicator.injectedView())
+            loadingIndicator.startAnimation()
+        } else {
+            addLoadingText(below: loadingIndicatorView)
+            loadingIndicatorView.startAnimating()
+        }
+    }
```

Comment:
@vizaknaidavid @zladzeyka  Could you tell me how custom loading indicator should work? Should we allow displaying both at the same time ? Or one replaces another?

Reply:
@chriskryn  there should be 1 loading indicator 😎: default or with a custom view.

Reply:
@vizaknaidavid there is no need to add a custom view for a loading indicator to a super view if giniConfiguration.analysisScreenLoadingIndicator is not set,

Reply:
@zladzeyka  Ok  :slightly_smiling_face: , I think it is used to center both indicator and text within the screen.

Reply:
@vizaknaidavid @zladzeyka So do we need a text under custom indicator ? Then I would think that the custom indicator should be enough .

Reply:
@zladzeyka I add a container that is centered in the screen in which I add the loading indicator and the text, so they are centered in the screen. If the custom indicator is not set, the conatiner will contain the default loading and the text.

Reply:
So can this thread be resolved without changes? @chriskryn  @zladzeyka  :D

Reply:
Yes, I think so ;)